### PR TITLE
Update office-365-groups.md

### PR DESCRIPTION
**Pull request by msbemba**
I have added other resources that a user cannot create when he is limited from creating groups. I can confirm that these cannot be created when there is a restriction, but I would like you to review and confirm if this can be added to the document.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5086

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/create-groups/office-365-groups.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Microsoft 365 Groups is the foundational membership service that drives all team
...
 
-Any user can create a group unless you [limit group creation to a specific set of people](../../solutions/manage-creation-of-groups.md). If you limit group creation, users who cannot create groups will not be able to create SharePoint sites, Planners, or teams. These services require the people creating them to be able to create a group. Users can still participate in group activities, such as creating tasks in Planner or using Teams chat, provided they are a member of the group.
+Any user can create a group unless you [limit group creation to a specific set of people](../../solutions/manage-creation-of-groups.md). If you limit group creation, users who cannot create groups will not be able to create SharePoint sites, Planners, teams, Outlook group calendar, Stream Groups, Yammer groups, Shared Library in OneDrive or a A Shared Power BI Workspace. These services require the people creating them to be able to create a group. Users can still participate in group activities, such as creating tasks in Planner or using Teams chat, provided they are a member of the group.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/create-groups/office-365-groups.md]:
```suggestion
Any user can create a group unless you [limit group creation to a specific set of people](../../solutions/manage-creation-of-groups.md). If you limit group creation, users who cannot create groups will not be able to create SharePoint sites, Planners, teams, Outlook group calendars, Stream groups, Yammer groups, Shared libraries in OneDrive, or shared Power BI workspaces. These services require the people creating them to be able to create a group. Users can still participate in group activities, such as creating tasks in Planner or using Teams chat, provided they are a member of the group.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT Please merge this PR. Thanks.

**Comment by MikePlumleyMSFT** [comment]
@ArunMSFT can you review?

**Comment by ArunMSFT** [comment]
Change looks good, please ensure it matches our style guide.

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT

**Pull request merged**